deploy/docker/pump_wrapper.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/sh
```

Comment:
#1202 is imminent: https://github.com/pump-io/pump.io/issues/1202#issuecomment-291050095

Please nuke this workaround :)